flow/scheduling/__init__.py

```diff
@@ -2,14 +2,3 @@
 # All rights reserved.
 # This software is licensed under the BSD 3-Clause License.
 """Defines the API for the scheduling system."""
```

Comment:
Is the intent here to completely remove schedulers from the public API? That seems dramatically out of scope for this PR.

Reply:
They're still in the public API -- but users must specify them with a full name: `flow.scheduling.torque.TorqueScheduler`. The reason I did this is that importing the Scheduler subclasses into the `scheduling` namespace (like `flow.scheduling.TorqueScheduler`) causes Sphinx to fail to find references to the right classes. The references would find `flow.scheduling.TorqueScheduler` but wouldn't link to the corresponding class docs. These classes were not mentioned in the documentation _at all_ before this PR, so while I recognize it's an API break, I'm not too worried about making this change. All internal code already used the full namespace and not the short form. If you know how to solve this issue without the API break, I'd be happy to learn more.

Reply:
*sigh OK yeah I have fixed this problem before but I don't recall how. Fiddling with Sphinx usually takes me 5 mins and a few tries to get just right. For now I'm OK with this solution, I agree that it's not too meaningful an API break. Since deprecating schedulers from the public API is something we've discussed recently I didn't want that to just happen without further discussion, but this particular change is fine with me for now.